source/common/router/retry_state_impl.cc

```diff
@@ -71,23 +71,19 @@ RetryStateImpl::RetryStateImpl(const RetryPolicy& route_policy, Http::HeaderMap&
   // Merge in the route policy.
   retry_on_ |= route_policy.retryOn();
   retries_remaining_ = std::max(retries_remaining_, route_policy.numRetries());
+  uint32_t base = runtime_.snapshot().getInteger("upstream.base_retry_backoff_ms", 25);
+  backoff_strategy_ptr_ = std::make_unique<JitteredBackOffStrategy>(base, base * 10, random_);
```

Comment:
Here I have added base * 10 as the maximum interval (bound). I think it should be sufficient. LMK if you think otherwise.

Reply:
Seems reasonable, maybe add a comment or constant.